cmd/libsnap-confine-private/string-utils.c

```diff
@@ -69,3 +70,48 @@ int sc_must_snprintf(char *str, size_t size, const char *format, ...)
 
 	return n;
 }
+
+char *sc_must_stpcpy(char *buf, size_t buf_size, char *dest, const char *src)
```

Comment:
I think this function is misnamed since the arg list doesn't match stpcpy() (note that sc_must_snprintf() matches snprintf()). With something named 'sc_must_stpcpy' I would expect it to be 'sc_must_stpcpy(char *dest, const char *src)`. Of course, it cannot be that cause you need to know the buf_size to make sure everything is ok.